packages/neuron-ui/src/components/NervosDAORecord/index.tsx

```diff
@@ -257,9 +260,7 @@ export const DAORecord = ({
         </span>
       </div>
 
-      <div className={styles.amount}>
-        <span>{`${shannonToCKBFormatter(capacity)} CKB`}</span>
-      </div>
+      <CopyZone className={styles.amount} content={amount.replace(/,/g, '')}>{`${amount} CKB`}</CopyZone>
```

Comment:
We may need a number formatter so that these replace regex can be re-used in different places.

Reply:
Yep, the method [shannon to ckb](https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-ui/src/utils/formatters.ts#L102) has the third param to specify the delimiter. I'll use update them in the future.